src/DotNet.Testcontainers/Containers/Configurations/Databases/OracleTestcontainerConfiguration.cs

```diff
@@ -29,6 +29,10 @@ public override string Password
     }
 
     public override IWaitForContainerOS WaitStrategy => Wait.ForUnixContainer()
-      .UntilPortIsAvailable(this.DefaultPort);
+      .UntilCommandIsCompleted("bin/bash","-c",
+        "export PATH=/u01/app/oracle/product/11.2.0/xe/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin;" +
+        "export ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe;" +
```

Comment:
I don't think **both** exports are necessary to test the database connection. Just use the absolute path to `sqlplus`.

Reply:
When i'm trying with absolute path i also have problems with export ORACLE_HOME;
![image](https://user-images.githubusercontent.com/47704348/77247939-d37e4600-6c46-11ea-91b3-a78219244597.png)

Reply:
Shouldn't that be part of the Docker image? `sqlplus` should already be in the `PATH` variable, otherwise the binary couldn't be found on other Oracle images.